terraform/shared/scripts/setup.sh

```diff
@@ -19,7 +19,7 @@ VAULTDOWNLOAD=https://releases.hashicorp.com/vault/${VAULTVERSION}/vault_${VAULT
 VAULTCONFIGDIR=/etc/vault.d
 VAULTDIR=/opt/vault
 
-NOMADVERSION=0.8.4
+NOMADVERSION=0.8.6
```

Comment:
you may want to bump the versions for Consul and Vault as well

Reply:
Good idea